_docs/reference/release-notes.md

```diff
@@ -1,13 +1,84 @@
+end-to-end overhead.
+
+- **Avoid Hot Restarts**. The need to hot-restart Envoy has been mostly eliminated through effective use of LDS/RDS/CDS/EDS.
+
+- **Reduced Memory Use**. Significantly reduced the size of the sidecar helper agent, from 50Mb to 7Mb.
+
+- **Improved Mixer Latency**. Mixer now clearly delineates configuration-time vs. request-time computations, which avoids doing extra setup work at 
+request-time for initial requests and thus delivers a smoother average latency. Better resource caching also contributes to better end-to-end perf.
+
+- **Reduced Latency for Egress Traffic**. We now forward traffic to external services directly from the sidecar.
+
+#### Traffic Management
+
+- **Egress Rules**. It’s now possible to specify routing rules for egress traffic.
+
+- **New Protocols**. Mesh-wide support for WebSocket connections, MongoDB proxying.
+
+- **Other Improvements**. Ingress properly supports gRPC services, interfacing headless services, better support for health checks, and
+Jaeger tracing.
+
+#### Policy Enforcement & Telemetry
+
+- **Ingress Policies**. Policies can now be applied to north-south traffic in addition to east-west traffic supported in 0.1.
+
+- **Support for TCP Services**. In addition to the HTTP-level policy controls available in 0.1, 0.2 introduces support for TCP-centric policy controls for 
+TCP services.
+
+- **New Mixer API**. The API that Envoy uses to interact with Mixer has been completely redesigned for increased robustness, flexibility, and to support 
+rich proxy-side caching and batching for increased performance.
+
+- **New Mixer Adapter Model**. A new adapter composition model makes it easier to extend Mixer by adding whole new classes of adapters via templates. This 
+new model will serve as the foundational building block for many features in the future.
+
+- **Improved Mixer Build Model**. It’s now easier to build a Mixer binary that includes custom adapters.
+
+- **Mixer Adapter Updates**. The built-in adapters have all been rewritten to fit into the new adapter model. The stackdriver adapter has been added for this
+release. The experimental redisquota adapter has been removed in the 0.2 release, but is expected to come back in production quality for the 0.3 release.
+
+- **Mixer Call Tracing**. Calls between Envoy and Mixer can now be traced and analyzed in the Zipkin dashboard.
+
+#### Security
+
+- **Mutual TLS for TCP Traffic**. In addition to HTTP traffic, mutual TLS is now supported for TCP traffic as well.
+
+- **Identity Provisioning for non-Kubernetes Nodes**. Auth supports a new mechanism using a per-node agent for identity provisioning. This agent runs on each
+node (VM / physical machine) and is responsible for generating and sending out the CSR (Certificate Signing Request) to get certificates from Istio CA. This is an early version of this feature which has limitations (such as requiring a flat network across containers and VMs).
```

Comment:
No matter auth is enabled or not, we will have this issue either way. So the limitation is not on security, it's on network reachability between VM and containers. Please consider document it somewhere else like mesh expansion.

Reply:
agreed with wattli. This is not a limitation for this identity provisioning feature. CSR doesn't actually require flat network. Lets remove "This is an early version ..." to avoid confusion

Reply:
i think the concern is that it is not clear this "Identity Provisioning for non-Kubernetes Nodes" is part of mesh expansion for the user.

Reply:
would "Identity Provisioning for VM" be clearer rather than "non-kubernetes nodes"

Reply:
Maybe add a new Mesh Expansion section and include a few items in there, with the caveat at the top instead of on the auth piece?